ONSdigital/rm-reporting
# Add responses dashboard endpoint

### What is the context of this PR?
Adds a new end point for the [responses dashboard](https://github.com/ONSdigital/sdc-responses-dashboard).

You can run `ras-rm-docker-dev` locally to get a valid collection exercise id for a live `SEFT` survey. Run acceptance tests to set up this data.

Use the following query to get the above for a `BRICKS` survey.
```
SELECT id, "name", periodstartdatetime, statefk, samplesize, exerciseref, user_description, created, updated, deleted, survey_uuid
FROM collectionexercise.collectionexercise
where collectionexercise."name" like 'Monthly Survey of%'
and collectionexercise.collectionexercise.statefk = 'READY_FOR_LIVE'
```

### How to review 
1. Ensure a valid response is returned for a valid collection exercise id.
2. Ensure a 400 error is returned for an invalid or malformed collection exercise id.
3. Also check unit tests are sufficient.
